runtime/Makefile

```diff
@@ -15,7 +15,7 @@ ENABLE_LIGHTNING_KOKKOS?=ON
 ENABLE_OPENQASM?=ON
 ENABLE_ASAN?=OFF
 # TODO: Update to v0.35.0 after the next lightning release.
-LIGHTNING_GIT_TAG_VALUE?="2716864521c539429bd382b92151a918d1a076ce"
+LIGHTNING_GIT_TAG_VALUE?="master"
```

Comment:
Going what with was said in the meeting? Let's not target master and target a specific commit. Just change to the specific commit in master for the time being.